python-backend/requirements.txt

```diff
@@ -11,34 +7,25 @@ Flask-APScheduler==1.11.0
 Flask-Caching==1.4.0
 Flask-Cors==3.0.6
 flask-compress==1.4.0
-Flask-Migrate==2.2.1
 flask-restplus==0.11.0
 Flask-SQLAlchemy==2.3.2
 flask_jwt_oidc==0.1.5
 GeoAlchemy2==0.5.0
-itsdangerous==0.24
-Jinja2==2.10
-jsonschema==2.6.0
 ldap3==2.6.0
-Mako==1.0.7
-MarkupSafe==1.0
-more-itertools==4.2.0
 names==0.3.0
-pluggy==0.7.1
 psycopg2==2.7.5
 psycopg2-binary==2.7.5
 py==1.5.4
 pytest==3.10.1
 pytest-cov==2.5.1
 python-dateutil==2.7.3
 python-dotenv==0.8.2
-python-editor==1.0.3
-pytz==2018.5
 redis==3.0.1
-requests==2.20.0
+requests==2.21.0
 six==1.11.0
-SQLAlchemy==1.2.10
+SQLAlchemy==1.3.3
 sqlalchemy-filters==0.8.0
+urllib3==1.25.1
```

Comment:
Needed by requests,  some other package is forcing an older version of urllib to be installed and requests seemed incompatible with that